x/programs/rust/examples/token/src/lib.rs

```diff
@@ -10,13 +17,19 @@ pub fn init(state: State) -> bool {
         .is_ok()
 }
 
-/// Gets total supply or -1 on error.
+/// @notice Gets total token supply or -1 on error
+/// @param state The current program state
```

Comment:
I don't think it's useful to continuously repeat the exact same definition of `State` when the struct itself should be clearly documented. The `public` macro should also be documented and linked. 

We don't need `@param` and `@returns` like you would in JavaScript.

Reply:
I will come up with a comment spec, this is just a start

Reply:
You don't really need a spec. If all your types are well documented, the built-in doc links pretty much handle everything you would want. The function signature itself partially serves as documentation.